maro/simulator/scenarios/data_center/cpu_reader.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from maro.data_lib.binary_reader import BinaryReader
+
+
+class CpuReader:
+
+    def __init__(self, data_path: str):
+        self._data_path = data_path
+        self._cpu_reader = BinaryReader(self._data_path)
+        self._cpu_item_picker = self._cpu_reader.items_tick_picker(
+            start_time_offset=self._cpu_reader.header.starttime,
```

Comment:
What's the difference between this and
```
            start_time_offset=0,
            end_time_offset=self._cpu_reader.header.endtime - self._cpu_reader.header.starttime,
```
?

Reply:
no difference if the start tick is 0.